lambdas/s3hash/src/t4_lambda_s3hash/__init__.py

```diff
@@ -372,25 +394,76 @@ async def compute_checksum_legacy(location: S3ObjectSource) -> Checksum:
     return Checksum.sha256(hashobj.digest())
 
 
+def combine_crc64nvme(part_crcs: T.List[bytes], part_sizes: T.List[int]) -> bytes:
```

Comment:
probably we could complete MPU to make it calculate it for us
but I think I had impression I saw it could be relatively slow

from other side code for combining CRC could be useful anyway